libs/sql-parser/src/main/java/io/crate/sql/tree/DefaultTraversalVisitor.java

```diff
@@ -222,8 +222,13 @@ protected R visitQuerySpecification(QuerySpecification node, C context) {
         if (node.getWhere().isPresent()) {
             node.getWhere().get().accept(this, context);
         }
-        for (Expression expression : node.getGroupBy()) {
-            expression.accept(this, context);
+        if (node.getGroupBy().isPresent()) {
+            GroupBy groupBy = node.getGroupBy().get();
+            if (!groupBy.isAll()) {
+                for (Expression expression : groupBy.getExpressions()) {
+                    expression.accept(this, context);
+                }
+            }
```

Comment:
```suggestion
            for (Expression expression : groupBy.getExpressions()) {
                expression.accept(this, context);
            }
```

If `isAll` is true, the expression list would be empty anyway